docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -181,139 +223,424 @@ spec:
+This configuration disables OpenSearch and enables the internal kubernetes Elasticsearch deployment:
+
+<details>
+<summary>Show configuration changes to disable external OpenSearch usage</summary>
+
+```yaml
+global:
+  elasticsearch:
+    enabled: true
+  opensearch:
+    enabled: false
+
+elasticsearch:
+  enabled: true
+```
+
+</details>
+
+##### Using internal PostgreSQL instead of the managed Aurora
+
+If you prefer not to use an external PostgreSQL service, you can switch to the internal PostgreSQL deployment. In this case, you will need to configure the Helm chart as follows and remove certain configurations related to the external database and service account:
+
+<details>
+<summary>Show configuration changes to disable external database usage</summary>
+
+```yaml
+identityKeycloak:
+  postgresql:
+    enabled: true
+
+  # Remove external database configuration
+  # externalDatabase:
+  #   ...
+
+  # Remove service account and annotations
+  # serviceAccount:
+  #   ...
+
+  # Remove extra environment variables for external database driver
+  # extraEnvVars:
+  #   ...
+
+webModeler:
+  # Remove this part
+
+  # restapi:
+  #     externalDatabase:
+  #         url: jdbc:aws-wrapper:postgresql://${DB_HOST}:5432/${DB_WEBMODELER_NAME}
+  #         user: ${DB_WEBMODELER_USERNAME}
+  #         existingSecret: webmodeler-postgres-secret
+  #         existingSecretPasswordKey: password
+
+identity:
+  # Remove this part
+
+  # externalDatabase:
+  #     enabled: true
+  #     host: ${DB_HOST}
+  #     port: 5432
+  #     username: ${DB_IDENTITY_USERNAME}
+  #     database: ${DB_IDENTITY_NAME}
+  #     existingSecret: identity-postgres-secret
+  #     existingSecretPasswordKey: password
+```
+
+</details>
+
+##### Fill your deployment with actual values
+
+Once you've prepared the `values.yml` file, use the `envsubst` command to substitute the environment variables with their actual values. Run the following command:
+
+```bash
+# generate the final values
+envsubst < values.yml > generated-values.yml
+
+# print the result
+cat generated-values.yml
+```
+
+Starting from **Camunda 8.6**, you need to store various passwords in a Kubernetes secret, which will be consumed by the Helm chart. Below is an example of how to set up the required secret:
+
+You can use `openssl` to generate random secrets and store them in environment variables:
+
+```bash reference
+https://github.com/camunda/camunda-tf-eks-module/blob/feature/opensearch-doc/examples/camunda-8.6/procedure/generate-passwords.sh
+```
+
+Next, use these environment variables in the `kubectl` command to create the secret.
+
+Note:
+
+- The values for `postgres-password` and `password` are not required if you are using an external database. If you choose not to use an external database, please provide those values.
+- The `smtp-password` should be replaced with the appropriate external value.
```

Comment:
smtp is just required for Web Modeler.
Could be considered optional and if kept, should maybe link to some related documentation. I was quite confused for a second where smtp was suddenly coming from.

Reply:
when you deploy the helm chart, it mentions only Web Modeler.

looking at the code: https://github.com/search?q=repo%3Acamunda%2Fcamunda-platform-helm+smtp-password&type=code

it also only reveals the Modeler.

Reply:
yeah the email connector - https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/email/

> Use Camunda secrets to avoid exposing your sensitive data as plain text. See [managing secrets](https://docs.camunda.io/docs/components/console/manage-clusters/manage-secrets/).

Reply:
in self-managed that would be via env variables.
I guess the helm chart does not automatically add it but would require a user to set it up accordingly.